browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -477,9 +477,9 @@ TEST_F(KeyringServiceUnitTest, CreateEncryptorForKeyring) {
   {
     KeyringService service(GetPrefs());
     EXPECT_TRUE(service.CreateEncryptorForKeyring("123", "default"));
-    EXPECT_NE(service.encryptor_, nullptr);
+    EXPECT_NE(service.encryptors_.at("default"), nullptr);
     EXPECT_TRUE(service.CreateEncryptorForKeyring("456", "keyring2"));
-    EXPECT_NE(service.encryptor_, nullptr);
+    EXPECT_NE(service.encryptors_.at("keyring2"), nullptr);
```

Comment:
need to check encryptor of default is not the same as keyring2
and `encoded_nonce != encoded_nonce2`

Reply:
done